doc/adrs/144_add_support_for_emf_models_migration.adoc

```diff
@@ -18,6 +18,10 @@ One important thing to consider is that BasicExtendedMetaData methods assume tha
 
 The version of the last migration used on a model will be stored at the root of the JSON document on EMFJson side.
 
+The version of a migration participant will represent when it was added to the project and will respect this structure : `ITERATION-YYYYMMDDHHMM`
+
+For example `2024.3.2-202404221400` for a participant added the 22-04 2024 at 14h00 during the iteration 2024.3.2.
```

Comment:
An iteration version can only end in `.0` so the version number would be `2024.5.0-202404221400` to indicate a migration participant contributed during the development of 2024.5.0